bin/tailscale-device-delete

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+# Delete a tailscale device by host name
+# Usage:
+#   `tailscale-device-delete factorio`
+set -euo pipefail
+
+hostname="$1"
+
+# Docs: https://github.com/tailscale/tailscale/blob/main/api.md
+api="https://api.tailscale.com/api/v2"
+# Using "simple" curl auth, so append a colon
+auth="$TAILSCALE_API_KEY:"
+
+# For users, tailnet ID is their email.
+url="$api/tailnet/$TAILSCALE_EMAIL/devices"
```

Comment:
Blocking: The tailnet organization depends on the authentication provider you use. My tailnet organization for example is `nikitawootten.github`. I suggest renaming this env var to `TAILSCALE_ORG`:
```suggestion
url="$api/tailnet/$TAILSCALE_ORG/devices"
```